jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/explorer/CodeWhispererServiceNode.kt

```diff
@@ -95,23 +97,21 @@ class CodeWhispererServiceNode(
     override fun update(presentation: PresentationData) {
         super.update(presentation)
         val connectionType = CodeWhispererExplorerActionManager.getInstance().checkActiveCodeWhispererConnectionType(project)
+
         when (connectionType) {
+            CodeWhispererLoginType.Expired -> {
+                presentation.addText(message("codewhisperer.explorer.root_node.login_type.expired"), SimpleTextAttributes.GRAY_ATTRIBUTES)
+            }
             CodeWhispererLoginType.Accountless -> {
                 presentation.addText(message("codewhisperer.explorer.root_node.login_type.accountless"), SimpleTextAttributes.GRAY_ATTRIBUTES)
             }
 
             CodeWhispererLoginType.SSO -> {
-                // Only show this hint text when CodeWhisperer is using secondary connection
-                if (isCodeWhispererUsingSecondaryConnection(project)) {
-                    presentation.addText(message("codewhisperer.explorer.root_node.login_type.sso"), SimpleTextAttributes.GRAY_ATTRIBUTES)
-                }
+                presentation.addText(message("codewhisperer.explorer.root_node.login_type.sso"), SimpleTextAttributes.GRAY_ATTRIBUTES)
```

Comment:
So now we will always show the hint text for AWS Builder ID/Identity Center?

Reply:
yes so it's more obvious that if they are connected or not